config/locales/en.yml

```diff
@@ -399,6 +403,20 @@ en:
           upheld: Upheld by ICO
           overturned: Overturned by ICO
         ico_decision_comment: "Details of ICO's decision"
+      case_overturned_foi:
```

Comment:
Can case_overturned_foi and case_overturned_sar inherit from each other? To stop the duplication

Reply:
Yes!